.eslintrc.js

```diff
@@ -2,5 +2,9 @@
  * @type {import('@types/eslint').Linter.BaseConfig}
  */
 module.exports = {
-  extends: ["@remix-run/eslint-config", "prettier"],
+  extends: [
+    "@remix-run/eslint-config",
+    "@remix-run/eslint-config/jest",
```

Comment:
This one's deprecated and `@remix-run/eslint-config/jest-testing-library` should be used instead.
Since it's not exported (yet) by the package, we need to wait for https://github.com/remix-run/remix/pull/2322 before we can use `@remix-run/eslint-config/jest-testing-library`